management/src/commands/update-identity.ts

```diff
@@ -0,0 +1,92 @@
+import { IdentityNotFoundError, InvalidValidationCodeError } from "../errors";
+import { checkVerificationCode } from "../queries/check-verification-code";
+import { getIdentity } from "../queries/get-identity";
+import { IdentityTypes, ManagementCredentials } from "../types";
+import { addIdentityToUser } from "./add-identity-to-user";
+import { markIdentityAsPrimary } from "./mark-identity-as-primary";
+import { removeIdentityFromUser } from "./remove-identity-from-user";
+
+type VerifyValidationCodeData = {
+  validationCode: string;
+  eventId: string;
+};
+
+type NewIdentityData = {
+  value: string;
+  type: IdentityTypes;
+  primary: string;
+};
+
+async function updateIdentity(
+  managementCredentials: ManagementCredentials,
+  userId: string,
+  verifyValidationCode: VerifyValidationCodeData,
+  newIdentity: NewIdentityData,
+  identityToUpdateId: string,
+): Promise<void> {
+  const { validationCode, eventId } = verifyValidationCode;
+  const { value, type, primary } = newIdentity;
```

Comment:
For `primary`, should that be something that the client pass to this function or something that is infered from the identity we want to update?